packages/babel-parser/test/fixtures/es2015/template/error-after-newline/output.json

```diff
@@ -2,7 +2,7 @@
   "type": "File",
   "start":0,"end":14,"loc":{"start":{"line":1,"column":0,"index":0},"end":{"line":3,"column":2,"index":14}},
   "errors": [
-    "SyntaxError: Numeric separators are not allowed inside unicode escape sequences or hex escape sequences. (2:5)"
+    "SyntaxError: Invalid escape sequence in template. (2:1)"
```

Comment:
Unfortunately here we loose the nice error message because we can't report the error immediately, but we now just store the position of the error.